tinkering.py

```diff
@@ -1,11 +1,4 @@
 from posit.connect import Client
 
 with Client() as client:
-    print(client.get("v1/users"))
-    print(client.users.get("f55ca95d-ce52-43ed-b31b-48dc4a07fe13"))
-
-    users = client.users
-    users = users.find(lambda user: user["first_name"].startswith("T"))
-    users = users.find(lambda user: user["last_name"].startswith("S"))
-    user = users.find_one(lambda user: user["user_role"] == "administrator")
-    print(user)
+    print(client.users.count())
```

Comment:
Do you want to commit this? (Or maybe gitignore the file?)

Reply:
It's probably time to delete it. I think we are past the bootstrapping phase